cmd/clair/main.go

```diff
@@ -55,13 +54,8 @@ func main() {
 		}
 		switch f.Name {
 		case "conf":
-			cf, err := os.Open(fv)
-			if err != nil {
-				golog.Fatalf("failed to open config file: %v", err)
-			}
-			defer cf.Close()
-			if err := yaml.NewDecoder(cf).Decode(&conf); err != nil {
-				golog.Fatalf("failed to decode yaml config: %v", err)
+			if err := cmd.LoadConfig(&conf, fv, true); err != nil {
```

Comment:
Thinking aloud; do we want strict to be false here? Like, is it more useful to know all the errors at once or try repeatedly?

Reply:
I think permissive for a "check" tool and "strict" for actually using the config works the best once it's a valid Config document, but yeah it'll throw odd errors before that.